test/widgets/action_sheet_test.dart

```diff
@@ -551,6 +554,21 @@ void main() {
       final expectedLink = narrowLink(store, narrow, nearMessageId: message.id).toString();
       check(await Clipboard.getData('text/plain')).isNotNull().text.equals(expectedLink);
     });
+
+    testWidgets('Full Url Testing', (tester) async {
+      final message = eg.streamMessage();
+      final narrow = TopicNarrow.ofMessage(message);
+      await setupToMessageActionSheet(tester, message: message, narrow: narrow);
+      await tapCopyMessageLinkButton(tester);
+      await tester.pump(Duration.zero);
+
+      final customAccount = eg.selfAccount.copyWith(id: 2, realmUrl: Uri.parse('https://chat.example'));
+      await setupToMessageActionSheet(tester, message: message, narrow: narrow, account: customAccount);
+      final customAccountStore = await testBinding.globalStore.perAccount(customAccount.id);
+
+      final expectedLink = narrowLink(customAccountStore, narrow, nearMessageId: message.id).toString();
```

Comment:
These lines are too long. Try to wrap them so that they are under 80 columns wide.

For example:
```diff
-     await setupToMessageActionSheet(tester, message: message, narrow: narrow, account: customAccount);
+     await setupToMessageActionSheet(
+       tester, message: message, narrow: narrow, account: customAccount);

```